examples/guides/mp-restful-webservice/src/main/docs/index.adoc

```diff
@@ -0,0 +1,487 @@
+in it.
+<3> Prepares the response as JSON.
+
+=== Add the code for the resource's endpoints
+. Returning the default message
++
+[source,java]
+----
+include::{greet-resource-class}[tag=getDefaultMessage]
+----
+<1> Indicates the HTTP method: `GET`.
+<2> Tells JAX-RS that this method returns JSON.
+
+. Returning the personalized greeting
++
+[source,java]
+----
+include::{greet-resource-class}[tag=getMessageWithName]
+----
+<1> Declares the path parameter which is the name to use for personalizing
+the returned message.
+<2> Indicates the HTTP method: `GET`.
+<3> Tells JAX-RS that this method returns JSON.
+<4> Triggers injection of the path parameter `name` as an argument to the method.
+
+. Setting a new greeting message
++
+[source,java]
+----
+include::{greet-resource-class}[tag=setGreeting]
+----
+<1> Identifies the path parameter for the new greeting text.
+<2> It's an HTTP `PUT`.
+<3> Tells JAX-RS that this method both consumes and produces JSON.
+<4> JAX-RS injects the new greeting from the path parameter as the method argument.
+
+=== Add an `Application` class
+JAX-RS looks for an `Application`. Create `GreetApplication.java`.
+You or your IDE will add these imports:
+[source,java]
+----
+include::{greet-application-class}[tags=javaImports;helidonImports]
+----
+
+[source,java]
+.GreetApplication.java
+----
+include::{greet-application-class}[tags=greetAppBody;!healthAdditionToGetClasses]
+----
+<1> Have JAX-RS create only one instance of this class.
+<2> No path prefix for this application.
+<3> Class must extend `javax.ws.rs.core.Application`.
+<4> Reports the resource classes in the application.
+
+=== Write the `Main` class
+You or your IDE will add these imports:
+[source,java]
+----
+include::{main-class}[tags=javaImports;helidonMPImports]
+----
+
+. Add the `startServer` method
++
+[source,java]
+----
+include::{main-class}[tag=startServer]
+----
+<1> Automatically reads server configuration from `microprofile-config.properties`
+and then starts the reactive web server.
+
+. Write a method to initialize logging
++
+[source,java]
+----
+include::{main-class}[tag=setupLogging]
+----
+<1> Loads logging config from `logging.properties`
+
+. Add or modify the `main` method
++
+[source,java]
+----
+include::{main-class}[tag=main]
+----
+
+== Build and Run
+include::{se-guide-adoc}[tag=buildAndRun]
+
+== Add Health Checks
+include::{se-guide-adoc}[tag=addHealthChecksIntro]
+
+=== Add a new JAX-RS resource class
+Create `HealthResource.java` to define the endpoints for checking activeness and readiness.
+You or your IDE will add these imports:
+[source,java]
+----
+include::{health-resource-class}[tag=javaxImports]
+----
+
+. Declare the class with these annotations:
++
+[source,java]
+----
+include::{health-resource-class}[tag=classDecl]
+----
+
+. Declare an injected instance field to refer to the greeting message resource. This
+is the only input to the activeness check in our simple implementation.
```

Comment:
STYLE: "to the activeness check" --> Consider rephrasing as "to check whether the service is active"